release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -527,6 +527,69 @@ If the following conditions are true, the feature is enabled by default:
+User-defined projects can now use remote write to send exemplar data scraped by Prometheus to remote storage.
+To use this feature, configure remote write using the `sendExemplars` option in the `RemoteWriteSpec` resource.
+For more information, see xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#remotewritespec[RemoteWriteSpec] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-15-monitoring-impoved-alert-querying-for-user-defined-projects"]
+==== Improved alert querying for user-defined projects
+Applications in user-defined projects now have API access to query alerts for application namespaces via the rules tenancy port for Thanos Querier. 
+You can now construct queries that access the `/api/v1/alerts` endpoint via port 9093 for Thanos Querier, provided that the HTTP request contains a `namespace` parameter.
+In previous releases, the rules tenancy port for Thanos Querier did not provide API access to the `/api/v1/alerts` endpoint.
+
+[id="ocp-4-15-monitoring-prometheus-updated-to-tolerate-jitters-at-scrape-time"]
+==== Prometheus updated to tolerate jitters at scrape time
+The default Prometheus configuration in the monitoring stack has been updated so that jitters are tolerated at scrape time.
+For monitoring deployments that have shown sub-optimal chunk compression for data storage, this update helps to optimize data compression, thereby reducing the disk space used by the time series database in these deployments.
+
+[id="ocp-4-15-monitoring-improved-staleness-handling-for-the-kubelet-service-monitor"]
+==== Improved staleness handling for the kubelet service monitor
+Staleness handling for the kubelet service monitor has been improved in order to ensure that alerts and time aggregations are accurate.
+This improved functionality is active by default and makes the dedicated service monitors feature obsolete.
+As a result, the dedicated service monitors feature has been disabled and is now deprecated, and setting the `DedicatedServiceMonitors` resource to `enabled` has no effect.
+
+[id="ocp-4-15-monitoring-improved-ability-to-troubleshoot-reports-of-tasks-failing"]
+==== Improved ability to troubleshoot reports of tasks failing
+The reasons provided when tasks fail in monitoring components are now more granular so that you can more easily pinpoint whether a reported failure originated in components deployed in the `openshift-monitoring` namespace or in the `openshift-user-workload-monitoring` namespace.
+If the Cluster Monitoring Operator (CMO) reports task failures, the following reasons have been added to identify where the failures originated:
+
+* The `PlatformTasksFailed` reason indicates failures that originated in the `openshift-monitoring` namespace.
+* The `UserWorkloadTasksFailed` reason indicates failures that originated in the `openshift-user-workload-monitoring` namespace.
```

Comment:
in previous versions, there is also explanation for `MultipleTasksFailed`, but removed now, is that expected?
`The `MultipleTasksFailed` reason indicates failures that originated in both core platform monitoring and user-defined projects.`

Reply:
yes, was removed because of: https://github.com/openshift/openshift-docs/pull/71003#discussion_r1484277019 and https://github.com/openshift/openshift-docs/pull/71003#discussion_r1484573918